xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -356,11 +372,18 @@
         <Parameter Name="result" Type="System.IO.Pipelines.ReadResult" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="result">The <see cref="T:System.IO.Pipelines.ReadResult" /></param>
+        <param name="result">When this method returns, this value is set to a <see cref="T:System.IO.Pipelines.ReadResult" /> instance that represents the result of the read call.</param>
```

Comment:
This param wasn't complete.

Reply:
```suggestion
        <param name="result">When this method returns <see langword="true" />, this value is set to a <see cref="T:System.IO.Pipelines.ReadResult" /> instance that represents the result of the read call. Otherwise, this value is set to <see langword="default" />.</param>
```

Reply:
Oh right! It slipped my mind that the method returned a boolean. Thanks.